source/deployment-considerations/colocation.txt

```diff
@@ -0,0 +1,101 @@
+.. _rm-colocation:
+
+==========
+Colocation
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
```

Comment:
I see that the page is titled colcation, but we don't define what that is anywhere on the page, and instead jump right into kind of more general recommendations for environments. Maybe we should add a brief introduction for what colocation is and why it's important?

Reply:
Nice addition. The context is clearer now.

Reply:
Do we really need a hierarchy here? "Deployment considerations" is a better page title than "Colocation".